browser/speedreader/speedreader_tab_helper.h

```diff
@@ -23,20 +23,43 @@ class SpeedreaderTabHelper
     : public content::WebContentsObserver,
       public content::WebContentsUserData<SpeedreaderTabHelper> {
  public:
+  enum class DistillState {
+    kNone,
+    kReaderMode,
```

Comment:
it is not clear what is the difference between these two. Might utilize better names or comments

Reply:
added better comments.